services/productService.js

```diff
@@ -4,6 +4,13 @@ const getProducts = async (offset, limit, gender) => {
     return await productDao.getProducts(offset, limit, gender);
 };
 
+const getProductDetailsByProductId = async (productId) => {
+    return await productDao.productDetails(productId);
```

Comment:
Dao 함수명 변경 시 해당 부분 변경도 고려해주세요!
```suggestion
    return await productDao.getProductDetails(productId);
```

Reply:
전부 수정 완료했습니다!